src/lib.rs

```diff
@@ -797,6 +798,8 @@ pub struct EguiContextQuery {
     pub render_target_size: &'static mut RenderTargetSize,
     /// [`Window`] component, when rendering to a window.
     pub window: Option<&'static mut Window>,
+    /// Cursor for the
```

Comment:
Cursor for, for the, the Cursor for, Cursor the, the for, Cursor the Cursor for the

Reply:
Thanks for pointing that out!

It's actually a non trivial change: The handling of the cursor is changed in bevy 0.15 ; in this PR I moved the initialization of the cursor to the user through `configure_cursor` :
https://github.com/mvlabat/bevy_egui/blob/3be113b82f8b7122b33f92467d20d2560e0d7722/examples/ui.rs#L56-L60

But I think this code snippet should live with bevy_egui (or bevy_something :thinking: ; but in the meantime we should be able to add it if not present.)